### ISSUE TITLE
Missing dependency: tkinter

### ISSUE THREAD

**Issue by alexander-bauer** [vesposito/easyucs#5]
Installing EasyUCS on a fresh CentOS 7 host and then invoking `easyucs.py` yields 
```
ModuleNotFoundError: No module named 'tkinter'
```
It looks like `tkinter` isn't available through the normal Pip installation route, though I'm not quite clear on why. I was able to resolve it on CentOS 7 by installing `python3-tkinter`.

**Comment by alexander-bauer** [comment]
If it were possible to make this and other classes of dependencies (Flask for the webserver, etc) optional in order to slim down headless installs, that would be even better.

**Comment by vesposito** [comment]
This tkinter dependency issue has already happened to us a few times. We think it is coming from one of our dependencies (probably one that deals with graphics) but we haven't been able to identify properly where the issue is.

Could you please share the entire Traceback? It would be nice to understand which dependency has this tkinter requirement, and handle that missing dependency directly there.

**Comment by alexander-bauer** [comment]
Hi @vesposito, this is running from my fork so the line numbers may be off, but here's a complete traceback.

```
Traceback (most recent call last):
  File "/usr/local/bin/easyucs", line 4, in <module>
    runpy.run_module('easyucs.easyucs', run_name='__main__')
  File "/usr/lib64/python3.6/runpy.py", line 208, in run_module
    return _run_code(code, {}, init_globals, run_name, mod_spec)
  File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.6/site-packages/easyucs/easyucs.py", line 14, in <module>
    from easyucs.device.device import UcsImc, UcsSystem, BlindUcs, GenericUcsDevice, UcsCentral
  File "/usr/local/lib/python3.6/site-packages/easyucs/device/device.py", line 26, in <module>
    from easyucs.inventory.manager import UcsImcInventoryManager, UcsSystemInventoryManager, UcsCentralInventoryManager
  File "/usr/local/lib/python3.6/site-packages/easyucs/inventory/manager.py", line 13, in <module>
    from easyucs.inventory.inventory import UcsImcInventory, UcsSystemInventory, UcsCentralInventory
  File "/usr/local/lib/python3.6/site-packages/easyucs/inventory/inventory.py", line 14, in <module>
    from easyucs.inventory.ucs.chassis import UcsImcSioc
  File "/usr/local/lib/python3.6/site-packages/easyucs/inventory/ucs/chassis.py", line 11, in <module>
    from easyucs.inventory.ucs.blade import UcsSystemBlade
  File "/usr/local/lib/python3.6/site-packages/easyucs/inventory/ucs/blade.py", line 11, in <module>
    from easyucs.inventory.ucs.adaptor import UcsSystemAdaptor
  File "/usr/local/lib/python3.6/site-packages/easyucs/inventory/ucs/adaptor.py", line 10, in <module>
    from easyucs.inventory.ucs.port import UcsImcAdaptorPort, UcsImcNetworkAdapterPort, UcsSystemAdaptorPort
  File "/usr/local/lib/python3.6/site-packages/easyucs/inventory/ucs/port.py", line 9, in <module>
    from easyucs.inventory.ucs.neighbor import UcsSystemLanNeighborEntry, UcsSystemSanNeighborEntry
  File "/usr/local/lib/python3.6/site-packages/easyucs/inventory/ucs/neighbor.py", line 9, in <module>
    from easyucs.draw.ucs.neighbor import UcsSystemDrawNeighbor
  File "/usr/local/lib/python3.6/site-packages/easyucs/draw/ucs/neighbor.py", line 8, in <module>
    from PIL import Image, ImageDraw, ImageTk, ImageFont
  File "/usr/local/lib64/python3.6/site-packages/PIL/ImageTk.py", line 34, in <module>
    import tkinter
ModuleNotFoundError: No module named 'tkinter'
```

It looks like PIL is to blame.

**Comment by vesposito** [comment]
Issue has been fixed in latest commit (c9af7a392fe23dd462e8a73ec516353e752f2263) with version 0.9.5 - Was importing ImageTk from PIL library but it was not used
